backend/models/core/Entity.js

```diff
@@ -60,6 +60,63 @@ class Entity {
         })); 
     }
 
+    static async findBySlug({entity_type_slug, slug}) {
+      const db = new DB();
+
+      const sql = `SELECT entities.id, entity_types.id, entity_types.name, entity_types.slug, entities.slug, 
```

Comment:
Properly align this part for code readability.
```suggestion
      const sql = SELECT entities.id, entity_types.id, entity_types.name, 
                  entity_types.slug, entities.slug, attributes.name, attributes.type, 
                  attributes.\`order\`, \`values\`.value_string, 
                  \`values\`.value_long_string, \`values\`.value_integer, 
                  \`values\`.value_datetime,  \`values\`.value_double                       
                  FROM entities
                  LEFT JOIN entity_types ON entities.entity_type_id = entity_types.id
                  LEFT JOIN attributes ON attributes.entity_type_id = entity_types.id
                  LEFT JOIN \`values\` ON values.entity_id = entities.id 
                  AND values.attribute_id = attributes.id
                  WHERE entity_types.slug = :entity_type_slug 
                  AND entities.slug = :slug
                  ORDER BY attributes.\`order\` ASC
                  `;
```